frontend/src/pages/Users/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC, useState, useEffect, SyntheticEvent } from "react";
+import "./index.scss";
+import { IUser } from "../../types/users";
+import apiAxios from "../../utils/apiAxios";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import RowEditDeleteChip from "../../components/RowEditDeleteChip";
+
+export const Users: FC = () => {
+    const [userData, setUserData] = useState<any>();
+    const [userSearchBy, setUserSearchBy] = useState<string>('');
+    const [userEditChipRowLocation, setUserEditChipRowLocation] = useState<number>(-1);
+
+    const handleSubmitSearch = (e: SyntheticEvent) => {
+      e.preventDefault();
+      console.log("search for", userSearchBy)
+    }
+
+    const handleRowMouseEnter = (e: any) => {
+      setUserEditChipRowLocation(e.target?.parentElement.id);
+    }
+
+    const handlerowMouseLeave = (e: any) => {
+      setUserEditChipRowLocation(-1);
+    }
+  
+    useEffect(() => {
+      const axiosResponse = apiAxios();
```

Comment:
consider using a state next time

Reply:
Do you mean keeping apiAxios in state? Can you explain this a bit further?

Reply:
Disregard, it's hard for me to explain this one without actually testing it myself.  I'll test it in my PR and see if I can explain better.